concerned-coyotes/earlyinternet/news/tasks.py

```diff
@@ -0,0 +1,29 @@
+from newsapi import NewsApiClient
+
+from .models import Article
+
+
+API_KEY = "c8a36b9a701d42ca854f7c31de866ba4"
```

Comment:
It's probably not a good idea to hardcode API keys into the code (and commit/push it to a public repo). It would have been nice to see a different way of managing secrets here as an exercise in the project.